sql/analyzer/resolve_column_defaults.go

```diff
@@ -723,19 +723,19 @@ func resolveColumnDefaultsOnWrapper(ctx *sql.Context, col *sql.Column, e *expres
 				err = sql.ErrInvalidColumnDefaultFunction.New(funcName, col.Name)
 				return false
 			}
-			if newDefault.IsLiteral() {
+			if newDefault.IsParenthesized() == false {
```

Comment:
something like `IsUnparenthesizedDatetimeFunc` might make sense? and make the code a bit more self-describing?